packages/tupaia-web/src/features/EntitySearch/SearchResults.tsx

```diff
@@ -105,15 +105,15 @@ export const SearchResults = ({ searchValue, onClose }: SearchResultsProps) => {
   return (
     <Container>
       <ScrollBody>
-        {searchResults.map(({ code, name }) => {
+        {searchResults.map(({ code, qualified_name }) => {
           return (
             <ResultLink
               button
               key={code}
               onClick={onClose}
               to={{ ...location, pathname: `/${projectCode}/${code}` }}
             >
-              {name}
+              {qualified_name}
```

Comment:
ah yes this makes sense - this was added by Ethan in there server and we never updated the frontend. good find! This is a super pedantic request but might as well get it to be consistent. We have a convention of snake case on the server and camel case on the front end and Ethan was supposed to do the conversion in the server but could you add that.

To do so you need add use something called `camelcaseKeys` in the return line of `EntitySearchRoute.ts` in tupaia-web-server, which will do the conversion. See `EntityAncestorsRoute.ts` as an example of the same thing. 

Then you will need to update this to camel case. eg. `qualifiedName`